Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/settings/ui/SettingsActivityPresenter.java

```diff
@@ -27,18 +31,22 @@
 
   private MenuTag menuTag;
   private String gameId;
+  private Context context;
+
+  private static final Set<String> modifiedSettings = new HashSet<>();
```

Comment:
Would it be possible to make this non-static? You could pass it to `saveSettings` as an argument.

Reply:
Yes, done.